AEPCampaignClassic/Tests/UnitTests/CampaignClassicTests.swift

```diff
@@ -0,0 +1,261 @@
+/*
+ Copyright 2022 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+
```

Comment:
List of test cases for your reference
![Screen Shot 2022-08-09 at 2 06 58 PM](https://user-images.githubusercontent.com/8909148/183761534-da9398cc-5bff-40e6-9130-4116471d54b7.png)